pom.xml

```diff
@@ -254,27 +254,13 @@
       </dependency>
     </dependencies>
   </dependencyManagement>
-  <pluginRepositories>
-    <!--
-    needed for accumulo-maven-plugin SNAPSHOT version
-    see PR apache/maven-apache-parent#1
-    -->
-    <pluginRepository>
-      <releases>
-        <enabled>false</enabled>
-      </releases>
-      <id>apache.snapshots</id>
-      <name>Apache Snapshot Repository</name>
-      <url>https://repository.apache.org/snapshots</url>
-    </pluginRepository>
-  </pluginRepositories>
```

Comment:
You don't actually need to remove this section yet... and probably shouldn't until maven-apache-parent merges in the referenced pull request, and releases a new version.

Reply:
I don't understand why there is a need to depend on snapshot versions of a plugin.  This seems like something that can make the build unstable.

Reply:
You shouldn't normally depend on snapshots. But leaving this section in the pom can help when, during development, you want or need to. This is why it was being added to the parent pom. An updated version of the parent pom just hasn't been released yet.

Reply:
Personally I dislike when the build pulls down some random snapshot version.  Would it make sense to put this in a profile?

Reply:
> Personally I dislike when the build pulls down some random snapshot version. Would it make sense to put this in a profile?

It's not entirely random. Maven does try to resolve the most recent snapshot. But, I get your point. That behavior, however, is already established for non-plugins in the Apache parent POM.

It doesn't matter to me if we remove this, since we don't currently need it. I would just get annoyed re-adding it if we needed it again in future.